internal/discord/command-handlers.go

```diff
@@ -28,9 +28,12 @@ var (
 // AddCommands registers the slash commands with Discord
 func (d *Discord) AddCommands(s *discordgo.Session, event *discordgo.Ready) {
 	fmt.Printf("Initializing Discord...\n")
-
+	foundTargetGuild := false
 	for _, discordGuild := range event.Guilds {
-
+		if discordGuild.ID != d.GuildID {
+			continue
+		}
+		foundTargetGuild = true
```

Comment:
this should be in an else block

Reply:
ah i meant just that line, but this approach works too. we'll need to revert the inverse and just do `if discordGuild.ID == d.GuildID`